### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by TpRedNinja**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Review by DeathHound6 — APPROVED** [review]

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -20707,4 +20707,15 @@
...
+    <Games>
+        <Game>Assassin's Creed Syndicate/Jack-the-Ripper<Game>
+    </Games>
+    <URLs>
+        <URL>https://raw.githubusercontent.com/TpRedNinja/Assassins-Creed-Syndicate/main/ACS.asl</URL>
+    </Urls>
+    <Type>Script</Type>
+    <Description>Auto Splitter for Assassins Creed Syndicate (by TpRedNinja)</Description>
+    <Website>https://github.com/TpRedNinja/Assassins-Creed-Syndicate/blob/main/ACS.asl</Website>
+    </Autospliter>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:20720]:
```suggestion
    <Autosplitter>
        <Games>
            <Game>Assassin's Creed Syndicate<Game>
        </Games>
        <URLs>
            <URL>https://raw.githubusercontent.com/TpRedNinja/Assassins-Creed-Syndicate/main/ACS.asl</URL>
        </Urls>
        <Type>Script</Type>
        <Description>Auto Splitter for Assassins Creed Syndicate (by TpRedNinja)</Description>
        <Website>https://github.com/TpRedNinja/Assassins-Creed-Syndicate/blob/main/ACS.asl</Website>
    </Autosplitter>
```

Comment (TpRedNinja) [reply LiveSplit.AutoSplitters.xml:20720]: so you want me to remove the /jack the ripper part? and thats all?

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml:20720]: You can simply click "Commit suggestion".

Comment (TpRedNinja) [reply LiveSplit.AutoSplitters.xml]: oh well i just edit it i just moved the jack the ripper part to the description as currently only works for that working on it working for base game.

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -20731,6 +20731,17 @@
         <Description>Auto Splitter for Obama (by CodeNameMeteor)</Description>
         <Website>https://github.com/CodeNameMeteor/Autosplitters/blob/main/Obama/ObamaAny.asl</Website>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Assassin's Creed Syndicate</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/TpRedNinja/Assassins-Creed-Syndicate/main/ACS.asl</URL>
+        </Urls>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:20740]:
```suggestion
        </URLs>
```

**Pull request merged**